### PR TITLE
LAYOUTS-619 Sylius channel parameter type

### PR DESCRIPTION
**Pull request by RandyCupic**

### CODE REVIEW

File: lib/Parameters/ParameterType/ChannelType.php [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+ */
+final class ChannelType extends ParameterType
+{
+    public static function getIdentifier(): string
+    {
+        return 'sylius_channel';
+    }
+
+    /**
+     * Configures the options for this parameter.
```
Comment (emodric) [inline lib/Parameters/ParameterType/ChannelType.php]: Superfluous PHPDoc.

File: lib/Parameters/ParameterType/ChannelType.php [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+    protected function getValueConstraints(ParameterDefinition $parameterDefinition, $value): array
+    {
+        return [
+            new Constraints\All(
+                [
+                    'constraints' => [
+                        new Constraints\Type(['type' => 'numeric']),
+                        new Constraints\GreaterThan(['value' => 0]),
+                        new SyliusConstraints\Channel(),
```
Comment (emodric) [inline lib/Parameters/ParameterType/ChannelType.php:38]: I suppose this constraint is implemented in `LAYOUTS-616-channel-locale-condition-types` branch?

**Review by emodric — APPROVED** [review]

File: lib/Parameters/ParameterType/ChannelType.php (hunk 2 above)
Comment (RandyCupic) [reply lib/Parameters/ParameterType/ChannelType.php:38]: @emodric Yes, that's why this PR is targeting the `LAYOUTS-616-channel-locale-condition-types` branch.

File: lib/Parameters/ParameterType/ChannelType.php (hunk 1 above)
Comment (RandyCupic) [reply lib/Parameters/ParameterType/ChannelType.php]: Fixed in https://github.com/netgen-layouts/layouts-sylius/pull/5/commits/d6a63a66aa31f607e9ba8833ff89ab11217a49c6

**Pull request merged**